Internal: Release/3.9.0

matipojo opened the pull request.

rami-elementor committed:
Internal: Added PHP Docs on how to use classes aliases (#19839)

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

rami-elementor committed:
Tweak: Button widget - responsive control for border radius [ED-7544] (#19884)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [x] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Tweak: Button widget - responsive control for border radius

## Description
An explanation of what is done in this PR

* Tweak: Button widget - responsive control for border radius

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [x] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #18914

Co-authored-by: Ariel Klikstein <ariel@elementor.com>

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

rodolphebertozzo committed:
Tweak: Add responsive option for caption to images widgets (#19852)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [ ] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

Co-authored-by: Ariel Klikstein <ariel@elementor.com>

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

rami-elementor committed:
Tweak: Add responsive capability to BG image opacity [ED-8344] (#19940)

## PR Checklist
- [x] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Tweak: Add responsive capability to BG image opacity

## Description
An explanation of what is done in this PR

* Tweak: Add responsive capability to BG image opacity

## Test instructions
This PR can be tested by following these steps:

**Sections:**

* Create a section and edit it.
* Style tab > Background overlay.
* Set a bg image > change the opacity.
* Change the Responsive Mode to Tablet/mobile.

**Columns:**

* Create a section and edit column.
* Style tab > Background overlay.
* Set a bg image > change the opacity.
* Change the Responsive Mode to Tablet/mobile.

**Containers:**

Same as sections and columns...

## Quality assurance

- [x] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

rami-elementor committed:
Tweak: Add responsive capability to BG image hover opacity [ED-8344] (#19947)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [x] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Tweak: Add responsive capability to BG image hover opacity

## Description
An explanation of what is done in this PR

* Tweak: Add responsive capability to BG image hover opacity

## Test instructions
This PR can be tested by following these steps:

* See #19940

## Quality assurance

- [x] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #19659

Co-authored-by: Ariel Klikstein <ariel@elementor.com>

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

maork-elementor committed:
Tweak: Make Elementor compatible to WebP [ED-7895] (#19558)

Via Performance Lab plugin

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

ramiy committed:
Tweak: Background video should have a background color [ED-8437] (#9885)

Sections and columns have have Classic background (color / image),
Gradient, Video or Slideshow.

The video background is missing a fallback background color displayed
while the video is loaded.

This PR added a background color control to the video bg.

Before:


![video-bg-before](https://user-images.githubusercontent.com/576623/70865005-1900e800-1f61-11ea-85fc-3b78f6d77ce6.png)

After:

![video-bg-after](https://user-images.githubusercontent.com/576623/70865010-228a5000-1f61-11ea-8250-b93e7645acde.png)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [x] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Background video should have a background color as fallback while the
video is loading

## Description
An explanation of what is done in this PR

* Background video should have a background color as fallback while the
video is loading

## Test instructions
This PR can be tested by following these steps:

* Edit any **Section**
* Click the **Style** tab
* In the **Background** area
* Set the **Background Type** to **Video**
* Choose the **Color** - it will be used ab fallback color while the
video is loading.

## Quality assurance

- [x] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

Co-authored-by: Ariel Klikstein <ariel@elementor.com>

rami-elementor committed:
Tweak: Consistent border radius units [ED-8436] (#20033)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [ ] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Tweak: Consistent border radius units

## Description
An explanation of what is done in this PR

* Tweak: Consistent border radius units

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [x] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

rami-elementor committed:
Tweak: Add `s` & `ms` units to transition-duration controls [ED-8427] (#20042)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [ ] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Tweak: Add `s` & `ms` units to transition-duration controls

## Description
An explanation of what is done in this PR

* Tweak: Add `s` & `ms` units to transition-duration controls

Before:


![t-d](https://user-images.githubusercontent.com/92088692/196048736-f98a49d5-6415-4142-8a21-405d814dcbd2.png)

After:


![t-d-new](https://user-images.githubusercontent.com/92088692/196048745-709bdeee-c5c7-4891-b6b0-105683139c95.png)

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

Co-authored-by: Ariel Klikstein <ariel@elementor.com>

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

arielk committed:
Internal: Text selection colors on app not correct [ED-3121] (#20065)

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

KingYes committed:
Tweak: Upgrade the autoprefixer package to better minify CSS files [ED-8404] (#20000)

CSS Assets:
```
before 5.3M    total
after 4.6M    total
```

matipojo committed:
Merge remote-tracking branch 'elementor/develop' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

KingYes committed:
Tweak: Remove duplicate queries on admin page [ED-8346] (#20098)

yotamselementor committed:
Update version

yotamselementor committed:
Revert "Update version"

This reverts commit 162bf37193fd65b6d2acebe027243c3f2296f51b.

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

rodolphebertozzo committed:
Tweak: Make responsive all instances for border radius (#19896)

Co-authored-by: Ariel Klikstein <ariel@elementor.com>

maork-elementor committed:
Internal: Fix conflicts from `release/3.8.0` (#20138)

Co-authored-by: louiswol94 <louiswol94@gmail.com>
Co-authored-by: Yakir Sitbon <kingyes1@gmail.com>

KingYes committed:
Internal: Fix conflict from `release/3.8.0`

KingYes committed:
Internal: Fix conflict from `release/3.8.0`

KingYes committed:
Internal: Fix package lock file (#20143)

KingYes committed:
Tweak: Added `dnt` param to Vimeo embed background and Video widget [ED-8191] (#20133)

Will fix: #13797, #13631

Nevoss committed:
Internal: Fix eslint issues (#20146)

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

maork-elementor committed:
New: Introducing Background images lazy load experiment [ED-7846] (#19706)

matipojo committed:
Merge remote-tracking branch 'elementor/release/3.8.0' into release/3.9.0

matipojo committed:
Merge remote-tracking branch 'elementor/release/3.9.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

StyleShit committed:
Fix: Pasting an element into the Preview Container throws a JS error [ED-8181] (#20118)

Used the route args instead of the unreliable `elementor.getCurrentElement()` method.

ramiy committed:
Fix: Separate "Default" and "None" values in "Border Type" [ED-8511] (#20145)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [x] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Fix: Separate "Default" and "None" values in "Border Type"

## Description
An explanation of what is done in this PR

* This PR fixes several bug. It changes the initial label to "Default".
In addition it adds the ability to set border none.

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [x] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes: #11565, #13328, #11723

rami-elementor committed:
Tweak: Add more units to border width controls in various elements [ED-8465] (#20037)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [ ] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Tweak: Add more units to border width controls in various elements

## Description
An explanation of what is done in this PR

* Tweak: Add more units to border width controls in various elements

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

StyleShit committed:
Fix: Can't drag widgets into a Column after it's being emptied [ED-8499] (#20152)

elementorbot committed:
Merge branch 'release/3.8.0' into release/3.9.0

matipojo committed:
Merge branch 'master' into release/3.9.0

louiswol94 committed:
Fix: Predefined container structure with default padding is not working [ED-8400] (#20171)

matipojo commented on core/kits/documents/tabs/theme-style-form-fields.php:
```diff
@@ -136,16 +136,14 @@ protected function register_tab_controls() {
-					$input_selector => 'transition: {{SIZE}}ms',
+				'size_units' => [ 's', 'ms' ],
+				'default' => [
+					'unit' => 'ms',
```
@rami-elementor BC? What will happen for the old setting?

rami-elementor replied:
The old selector had hard-coded unit `{{SIZE}}ms`. For BC, I replaced the selector with `{{SIZE}}{{UNIT}}` and added a default of `'unit' => 'ms'`.

matipojo commented on core/kits/documents/tabs/theme-style-images.php:
```diff
@@ -200,16 +200,14 @@ protected function register_tab_controls() {
-					],
+				'size_units' => [ 's', 'ms' ],
+				'default' => [
+					'unit' => 's',
```
@rami-elementor BC?

rami-elementor replied:
The old selector had hard-coded unit `{{SIZE}}s`. For BC, I replaced the selector with `{{SIZE}}{{UNIT}}` and added a default of `'unit' => 's'`.

matipojo commented on includes/widgets/video.php:
```diff
@@ -1084,6 +1084,10 @@ public function get_embed_params() {
 
 			$params['autopause'] = '0';
+
+			if ( ! empty( $settings['yt_privacy'] ) ) {
+				$params['dnt'] = 'true';
+			}
```
@KingYes Maybe it should be here:
https://github.com/elementor/elementor/blob/master/assets/dev/js/frontend/handlers/video.js#L114

TzviRabinovitch committed:
Fix: First container handle is not centered on an RTL page  [ED-7788] (#20036)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [x] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [ X] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [X] I have tested this code to the best of my abilities
- [X] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

udidol committed:
Fix: Repeater controls in "non-content" tabs cause JS errors when running Copy-Paste Style [ED-8548] (#20199)

## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [X] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Fix: Repeater controls in "non-content" tabs cause JS errors when
running Copy-Paste Style

## Description
An explanation of what is done in this PR

* Fixes #19895

Nevoss committed:
New: Kit Elements Default Values [ED-8357] (#20113)

Co-authored-by: StyleShit <32631382+StyleShit@users.noreply.github.com>

maork-elementor committed:
Fix: Background appearing around widgets when using the Lazy load experiment [ED-8564] (#20225)

maork-elementor committed:
Fix: Image thumbnails glitch on front when using the Lazy Load experiment [ED-8563] (#20233)

matipojo committed:
Merge remote-tracking branch 'origin/develop' into release/3.9.0

rami-elementor committed:
Tweak: Replace the remaining 'Go Pro' labels [ED-8566] (#20204)

maork-elementor committed:
Internal: PHPUnit fails - Image HTML was changed (WebP) [ED-8586] (#20266)

TzviRabinovitch committed:
Tweak: Adjusted Dark Mode in Navigator when using Container [ED-7578] (#20190)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [ ] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [X] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [X] I have tested this code to the best of my abilities
- [X] I have added unittests to verify the code works as intended - minor
fix
- [x] Docs have been added / updated (for bug fixes / features)

Fixes #

TzviRabinovitch committed:
Merge remote-tracking branch 'elementor/develop' into release/3.9.0

rami-elementor committed:
Tweak: Remove unneeded CSS vendor prefixes [ED-8633] (#20287)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [x] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Tweak: Remove unneeded CSS vendor prefixes

## Description
An explanation of what is done in this PR

* CSS vendor prefixes should be covered by “**autoprefixer**” and based
on the `.browserslistrc` file settings. We should not hardcode vendor
prefixes in our `.scss` files.

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #16330
Related: #20000

Nevoss committed:
Internal: Elements Regression - Media Control + Widget Image  [ED-8279] (#19886)

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.9.0

StyleShit committed:
Tweak: Changed the confirmation modal content in Save as Default experiment [ED-8624] (#20286)

rami-elementor committed:
Tweak: Update recommended PHP version in system-info [ED-8567] (#20205)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [ ] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Tweak: Update recommended PHP version in system-info

## Description
An explanation of what is done in this PR

* Tweak: Update recommended PHP version in system-info

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

KingYes committed:
Tweak: Added `rel="preconnect"` tag to Google Fonts to improve performance [ED-8054] (#20251)

ref: https://github.com/elementor/elementor/issues/10886

rami-elementor committed:
Tweak: Escape translation strings for safe use in HTML output [ED-8565] (#20188)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [x] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Tweak: Escape translation strings for safe use in HTML output

## Description
An explanation of what is done in this PR

* We usually escape translation strings in our code. I found (and fixed)
40 translation strings that were not escaped. It's basically for
consistency in our code.

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

davseve committed:
Internal: Add UTM params to kit GoPro link URL in main screen [ED-8547]  (#20202)

rami-elementor committed:
Tweak: Remove redundant labels from group controls [ED-8558] (#20163)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [x] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Tweak: Remove redundant labels from group controls

## Description
An explanation of what is done in this PR

* When we use group controls, we don't set the `label` argument. For
consistency, I removed labels from a few more places where we did used
those labels. Removing them, does not affect the control label in the
editor. I tested it.

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [x] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #11666

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.9.0

TzviRabinovitch committed:
Internal: Playwright timeout fix (#20291)

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.9.0

rami-elementor committed:
Tweak: Add angle units to rotate/skew controls in various elements [ED-8449] (#20081)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [x] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Tweak: Add angle units to rotate/skew controls in various elements

## Description
An explanation of what is done in this PR

* Tweak: Add angle units to rotate/skew controls in various elements

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

rami-elementor committed:
Tweak: Add new styling controls to Post/Page Settings [ED-8483] (#20181)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [x] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Tweak: Add new styling controls to Post/Page Settings

## Description
An explanation of what is done in this PR

Before:


![page-body-margin-padding](https://user-images.githubusercontent.com/92088692/198867282-f935444d-8204-4eac-b645-6e26feffe025.png)

After:


![the-new-page-body-style](https://user-images.githubusercontent.com/92088692/198867286-3ca14bf3-7535-40b1-9053-e53152d4f683.png)

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [x] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

Nevoss committed:
Internal: Elements regression - Update font size refs (#20288)

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.9.0

elementorbot committed:
Merge remote-tracking branch 'origin/master' into develop

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.9.0

KingYes committed:
Tweak: Update privacy mode control description about Vimeo support (#20308)

Continue from #20133

StyleShit committed:
Tweak: Allow saving and reloading while in-place editing of documents [ED-8477] (#20150)

Nevoss committed:
Internal: Elements Regression - Create dependencies for controls [ED-8293] (#20316)

Nevoss committed:
New: Kit defaults - Import/Export runner [ED-8179] (#20099)

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.9.0

Nevoss committed:
Fix: Kits Defaults - Support for inner section [ED-8585] (#20261)

gerasimovdaniel committed:
Fix: PHP errors caused by deprecation module [ED-8650] (#20063)

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.9.0

StyleShit committed:
Internal: Added tests for deprecation module [ED-8650] (#20327)

elementorbot committed:
Bump 3.9.0-beta1

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.9.0

hein-obox committed:
Fix: Align items controls are not working (...) [ED-8382] (#20147)

matipojo committed:
Internal: Playwright timeout adjustments (#20337)

udidol committed:
Internal: Skip Container Screenshot Playwright tests (#20339)

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.9.0

matipojo committed:
Internal: Playwright - Skip unstable tests [ED-8679] (#20340)

rami-elementor committed:
Tweak: Add missing default unit for controls using angel units [ED-8683] (#20344)

Fixes:
https://github.com/elementor/elementor/commit/2aff735f740ecbd3d329758926ce4979482d8f05#r89341052
